docs/src/introduction.md

```diff
@@ -14,6 +14,14 @@ The architecture describes a theoretical upper bound of 710 thousand transaction
 
 All claims, content, designs, algorithms, estimates, roadmaps, specifications, and performance measurements described in this project are done with the author's best effort. It is up to the reader to check and validate their accuracy and truthfulness. Furthermore, nothing in this project constitutes a solicitation for investment.
 
+## Why Solana?
```

Comment:
This content was in the readme. Not sure where best to put it.  "Why Solana?" doesn't seem quite right.  Suggestions?

Reply:
Possibly just remove.

I'd move it to the end of the document for now.

The formal (e.g. for SEC compliance) statements look different. They are usually at the end of a document.